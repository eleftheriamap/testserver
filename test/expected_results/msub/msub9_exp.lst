
test/exp/test/gen/msub/msub9_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	d130ddce 	sub	x14, x14, #0xc37
   4:	d173bef7 	sub	x23, x23, #0xcef, lsl #12
   8:	116688a5 	add	w5, w5, #0x9a2, lsl #12
   c:	9b1795d3 	msub	x19, x14, x23, x5
  10:	8a000000 	and	x0, x0, x0
