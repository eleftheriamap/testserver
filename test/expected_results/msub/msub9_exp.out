Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 00000000009a2000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = fffffffffffff3c9
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = fffffff620e49000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = ffffffffff311000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0xd130ddce
0x00000004 : 0xd173bef7
0x00000008 : 0x116688a5
0x0000000c : 0x9b1795d3
0x00000010 : 0x8a000000
